cypress/integration/unit_tests/manage_disaster_test.js

```diff
@@ -352,6 +443,66 @@ describe('Unit tests for manage_disaster.js', () => {
                       .get())
         .then((doc) => expect(doc.exists).to.be.false);
   });
+
+  /**
+   * Sets up fresh asset data (no feature collections), and prepares fake page
+   * with damage/map-bounds-related HTML elements.
+   * @return {Cypress.Chainable<Document>}
+   */
+  function setAssetDataAndCreateDamageInputsForScoreValidationTests() {
+    const currentData = createDisasterData(['NY']);
+    currentData['asset_data']['map_bounds_sw'] = '0, 1';
+    disasterData.set(getDisaster(), currentData);
+    return cy.document().then((doc) => {
+      // Lightly fake out jQuery so that we can use Cypress selectors. Might not
+      // work if manage_disaster.js starts doing fancier jQuery operations.
```

Comment:
Why not just use jquery selectors here and below?

Reply:
Make buttons gray when disabled, and make this button not capitalize, since it has a lot of text in it. Hide the button until all asset retrieval is done.

There's a subtle issue here if Firestore's value for the asset for some score factor is not present in EE. Currently, we won't display that value in the select, but we don't overwrite the Firestore value or our local value until the user actually changes the dropdown, so we could incorrectly think the user had specified all assets, even though one doesn't exist. Not sure if worth addressing, since should be pretty hard for user to do that (easiest way would be to delete an asset, reload page, then not notice a select wasn't selected anymore). Not new in this PR either, although potentially more confusing because page will say we're good to go. Since we'll always be vulnerable to user deleting assets and not reloading page (in which case we'll show stale data forever), I think we can punt.